mapeditor/mainwindow.cpp

```diff
@@ -206,6 +194,35 @@ MainWindow::MainWindow(QWidget* parent) :
 	loadTranslation();
 
 	ui->setupUi(this);
+
+	setWindowIcon(QIcon{":/icons/menu-game.png"});
+	ui->toolBrush->setIcon(QIcon{":/icons/brush-1.png"});
+	ui->toolBrush2->setIcon(QIcon{":/icons/brush-2.png"});
+	ui->toolBrush4->setIcon(QIcon{":/icons/brush-4.png"});
+	ui->toolLasso->setIcon(QIcon{":/icons/brush-3.png"});
+	ui->toolLine->setIcon(QIcon{":/icons/brush-7.png"});
+	ui->toolArea->setIcon(QIcon{":/icons/brush-5.png"});
+	ui->toolFill->setIcon(QIcon{":/icons/brush-6.png"});
```

Comment:
these icons deserve a better name :)

Reply:
I agree! :) Should I rename the original files accordingly to a ui element these are used at?

Reply:
for sure. Icon name should relate to icon content.